test/scales/modifiedLogScaleTests.ts

```diff
@@ -2,187 +2,241 @@
+        let a = [-100, -10, -3, 0, 1, 3.64, 50, 60, 200];
+        let b = a.map((x) => scale.scale(x));
+        assert.deepEqual(b, b.slice().sort().reverse(), "should be decreasing function; reverse is sorted");
+      });
 
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), requestedDomain, "calling autoDomain() overrides domainMin()");
+    });
 
-      let minEqualTop = scale.domain()[1];
-      scale.domainMin(minEqualTop);
-      assert.deepEqual(scale.domain(), [minEqualTop, minEqualTop * base],
-        "domain is set to [min, min * base] if the requested value is >= autoDomain()-ed max value");
+    describe("Other bases", () => {
+      it("uses 10 as the default base", () => {
+        let scale = new Plottable.Scales.ModifiedLog();
+        assert.strictEqual((<any>scale)._base, 10, "10 is the default base");
+      });
 
-      scale.domainMin(minInMiddle);
-      let requestedDomain2 = [-10, 10];
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => requestedDomain2);
-      assert.deepEqual(scale.domain(), [minInMiddle, requestedDomain2[1]], "adding another ExtentsProvider doesn't change domainMin()");
+      it("works with base 2", () => {
+        let scale = new Plottable.Scales.ModifiedLog(2);
+        assert.strictEqual((<any>scale)._base, 2, "base 2 is selected");
+        scale.domain([0, 16]);
+        scale.range([0, 1]);
+
+        assert.strictEqual(scale.scale(-2), -0.25, "scales negative values");
+        assert.strictEqual(scale.scale(0), 0, "scales 0");
+        assert.strictEqual(scale.scale(2), 0.25, "scales base");
+        assert.strictEqual(scale.scale(4), 0.5, "scales random values");
+        assert.strictEqual(scale.scale(16), 1, "scales maximum value");
+        assert.strictEqual(scale.scale(256), 2, "scales values outside the domain");
+
+        assert.strictEqual(scale.invert(1), 16, "inverts maximum value");
+        assert.strictEqual(scale.invert(0), 0, "inverts zero");
+      });
     });
 
-    it("domainMax()", () => {
-      let scale = new Plottable.Scales.ModifiedLog(base);
-      scale.padProportion(0);
-      let requestedDomain = [-5, 5];
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => requestedDomain);
+    describe("Auto Domaining", () => {
+      let scale: Plottable.Scales.ModifiedLog;
+      let base = 10;
 
-      let maxAboveTop = 10;
-      scale.domainMax(maxAboveTop);
-      assert.deepEqual(scale.domain(), [requestedDomain[0], maxAboveTop], "upper end of domain was set by domainMax()");
+      beforeEach(() => {
+        scale = new Plottable.Scales.ModifiedLog(base);
+        scale.padProportion(0);
+      });
 
-      let maxInMiddle = 0;
-      scale.domainMax(maxInMiddle);
-      assert.deepEqual(scale.domain(), [requestedDomain[0], maxInMiddle], "upper end was set even if requested value cuts off some data");
+      it("expands single value domains to [value / base, value * base].sort()", () => {
```

Comment:
Can we have a less codified way to word this?  Why do we want to expand the single value to this?